packages/@aws-cdk/aws-appmesh/test/test.gateway-route.ts

```diff
@@ -178,4 +179,97 @@ export = {
     test.equal(gatewayRoute.virtualGateway.mesh.meshName, meshName);
     test.done();
   },
+  'Can grant an identity all read permissions for a given GatewayRoute'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const mesh = new appmesh.Mesh(stack, 'mesh', {
+      meshName: 'test-mesh',
+    });
+    const gateway = new appmesh.VirtualGateway(stack, 'testGateway', {
+      mesh: mesh,
+    });
+    const myService = new appmesh.VirtualService(stack, 'vs-1', {
+      virtualServiceProvider: appmesh.VirtualServiceProvider.none(mesh),
+      virtualServiceName: 'target.local',
+    });
+
+    const gr = gateway.addGatewayRoute('gateway-http-route', {
+      routeSpec: appmesh.GatewayRouteSpec.http({
+        routeTarget: myService,
+      }),
+      gatewayRouteName: 'gateway-http-route',
+    });
+
+    // WHEN
+    const user = new iam.User(stack, 'test');
+    gr.grantRead(user);
+
+    // THEN
+    expect(stack).to(haveResourceLike('AWS::IAM::Policy', {
+      PolicyDocument: {
+        Statement: [
+          {
+            Action: [
+              'appmesh:DescribeGatewayRoute',
+              'appmesh:ListGatewayRoute',
+            ],
+            Effect: 'Allow',
+            Resource: {
+              Ref: 'testGatewaygatewayhttprouteD65B806A',
+            },
+          },
+        ],
+      },
+    }));
+
+    test.done();
+  },
```

Comment:
Empty line between the tests pliz, thanks!

Same comment in the other test files.